### DISCUSSION TITLE
Is 0th order SRM in your example the same thing as this?

### DISCUSSION THREAD

**Discussion by AreUTheDoctor** [jeshraghian/snntorch#99] [General]
This as for: http://www.scholarpedia.org/article/Spike-response_model#Spike_Response_Model_SRM.5C.28_0.5C.29

**Comment by jeshraghian** [comment]
Not exactly, but what is in that link is general enough to model the same dynamics of SRM0, just using a kernel-based approach instead of a recurrent method. Because of that difference, SRM0 has been changed to Alpha (as the membrane potential follows an alpha function).
